src/components/RiskTrackerUtil.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+export const spreadsheetUrl =
```

Comment:
This looks like a good place to put a list of past versions as well! Keeps the # of files that need to be changed when upgrading the spreadsheet small.

Reply:
(OK to do this in a follow on)

Reply:
added a comment with a link to https://docs.google.com/document/d/1iwTFoCS8lOIWWm-ZzcMZ_mPHgA8tHVVA3yhKY23gDu8/edit#